ImageManager/SerializationModels.py

```diff
@@ -21,8 +22,35 @@ def remove_null_values(self, data):
 
 image_schema = ImageSchema()
 
+ALLOWED_EXTENSIONS = set(['hex'])
 
-def parse_payload(request, schema):
+
+def allowed_file(filename):
+    return '.' in filename and \
+           filename.rsplit('.', 1)[1].lower() in ALLOWED_EXTENSIONS
+
+
+def parse_form_payload(request):
+    # Validate http header info
+    content_type = request.headers.get('Content-Type')
+    if (content_type is None) or (
+            "multipart/form-data" not in content_type) or not request.files:
+        raise HTTPRequestError(400, "Payload must be valid multipart/form-data")
```

Comment:
Shouldn't this be `application/octet-stream`? (I'm  assuming this is where we'd parse the software image itself)

Reply:
Honestly, I don't know, both curl and python requests treat it as multipart/form-data by default when sending files, but application/octet-stream does seem more apropriate